lib/html/pipeline/emoji_filter.rb

```diff
@@ -71,7 +71,42 @@ def asset_path(name)
 
       # Build an emoji image tag
       def emoji_image_tag(name)
-        "<img class='emoji' title=':#{name}:' alt=':#{name}:' src='#{emoji_url(name)}' height='20' width='20' align='absmiddle' />"
+        "<img #{img_html_attrs(name)}>"
```

Comment:
Good point. I missed that.

* Could you update the documentation at the top of the class explaining this new context option?
* What do you think about simplifying the methods? The new methods are not part of the public interface, and I feel they add more layers than necessary.  What about doing this new work in the existing `emoji_image_tag` method?